api/specs.proto

```diff
@@ -358,8 +358,18 @@ message ContainerSpec {
 
 	// CapabilityAdd sets the list of capabilities to add to the default capability list
 	repeated string capability_add = 27;
-	// CapabilityAdd sets the list of capabilities to drop from the default capability list
+	// CapabilityDrop sets the list of capabilities to drop from the default capability list
```

Comment:
Whoops! looks like we missed this in https://github.com/docker/swarmkit/pull/2965)

I guess the linter did not complain about this because it skips the generated `.pb.go` files 🤔 

Could you perhaps create a separate PR for this change?

Reply:
I reverted the change here and created #2968 :wink: